tools/train.py

```diff
@@ -168,7 +168,8 @@ def main():
         # checkpoints as meta data
         cfg.checkpoint_config.meta = dict(
             mmdet_version=__version__ + get_git_hash()[:7],
-            CLASSES=datasets[0].CLASSES)
+            CLASSES=datasets[0].CLASSES,
+            PALETTE=datasets[0].PALETTE)
```

Comment:
Is it necessary to record the palette in the checkpoint?
Any reason?

Reply:
This is the pattern in mmsegmentation. In `show_result` function, mmsegmentation will load palette from the model if no palette is specified.